mlir/lib/TableGen/Pattern.cpp

```diff
@@ -774,10 +830,23 @@ void Pattern::collectBoundSymbols(DagNode tree, SymbolInfoMap &infoMap,
       if (!treeArgName.empty() && treeArgName != "_") {
         DagLeaf leaf = tree.getArgAsLeaf(i);
 
-        // In (NativeCodeCall<"Foo($_self, $0, $1, $2)"> I8Attr:$a, I8:$b, $c),
+        // In (NativeCodeCall<"Foo($_self, $0, $1, $2)"> I8Attr:$a, I8:$b, $c,
+        // I8Prop:$d),
```

Comment:
Nit: add spacing so that I8Prop is intended more / inside NativeCodeCall).

Wouldn't we need a 4th argument to Foo here?